packages/create-universal-app/bin/generate.js

```diff
@@ -38,7 +38,18 @@ const runAsyncProcess = async (spawnFunction) => {
     }
 
     const cloning = ora(`Cloning repository to "${projectDir}"...`).start();
-    await runAsyncProcess(() => spawn("git", ["clone", repoURL, projectDir]));
+    await runAsyncProcess(() =>
+      spawn("git", [
+        "clone",
+        "--depth",
+        "1",
```

Comment:
Do you later destroy the git history and re-init?

Otherwise you may run into issues
<img width="700" alt="image" src="https://github.com/theodo-group/universal-app/assets/32868346/153fb35c-9dff-4ad8-a610-2fbc4b5db076">

Reply:
I would recommend destroying git history and re initing either way btw

Feels cleaner for the end user